src/fides/api/task/graph_task.py

```diff
@@ -733,6 +733,7 @@ def get_cached_data_for_erasures(
     value_dict = cache.get_encoded_objects_by_prefix(
         f"PLACEHOLDER_RESULTS__{privacy_request_id}"
     )
+    # TODO: Determine if this requires handling as well
```

Comment:
@seanpreston this is the main question I still had but wanted to get checks running on the PR. If needed, should be very easy to implement 👍🏽

Reply:
hmm, it _seems_ like it does, because i think we're also referencing the collection address when storing the placeholder results, so presumably we need to handle the same edge case? https://github.com/ethyca/fides/pull/3688/files#diff-f1bb24bdf53cb62b5d5ab785e4cda217afa21e57780fbc2dbd04ff674ee4fafbR134

Reply:
Yes, the same edge case needs to be handled here! What would be an issue for access requests would be an issue for erasure requests.

Reply:
Thank you both! Will get that sorted in here for both then 👍🏽

Reply:
Added in https://github.com/ethyca/fides/pull/3688/commits/fe4d1a7bd32607938ac1871619db3f50e58d7627

Reply:
Updated in https://github.com/ethyca/fides/pull/3688/commits/b57bf1e08fbfca7693d5eb2ed091609e50464a39 due to differences in the structure of the string. Parameterized the number of strings we strip from the front, hope that feels ok